### DISCUSSION TITLE
Gk0Wk/sidebar-resizer

### DISCUSSION THREAD

**Discussion by giscus** [tiddly-gittly/TiddlyWiki-CPL#48] [Plugins]
# Gk0Wk/sidebar-resizer

https://tw-cpl.netlify.app/

**Comment by gwankyun** [comment]
CPL版本不完整以致無法使用，安裝完成後僅有以下預設條目：
- $:/plugins/Gk0Wk/template/LICENSE
- $:/plugins/Gk0Wk/template/readme.tid

拖動安裝包含以下預設條目：
- $:/plugins/Gk0Wk/sidebar-resizer/hook.js
- $:/plugins/Gk0Wk/sidebar-resizer/LICENSE
- $:/plugins/Gk0Wk/sidebar-resizer/readme
- $:/plugins/Gk0Wk/sidebar-resizer/resizer
- $:/plugins/Gk0Wk/sidebar-resizer/style.css
